DRAFT_slitasje.Rmd

```diff
@@ -0,0 +1,1496 @@
+    set](#ANO-import)
+
+2.  Identify the [relevant](#naturtype) nature types and [subset](#NTM)
+    the data
+
+3.  Covert [ANO points to polygons](#ANO-points-to-poly)
+
+4.  For each polygon, extract the NiN-variable that has the lowest value
+    (worst-rules rule), e.g. like Fig. \@ref(fig:slitasje-naturetype).
+
+5.  [Combine](#combine-nt-ano) data sets
+
+6.  [Scale](#scale-slitasje-ind) the `slitasje` variable based on
+    reference values.
+
+7.  Define [homogeneous impact areas](#HIA) (HIA)
+
+8.  [Aggregate and spread](#spread-slitasje) indicator value across HIAs
+    and assessment regions
+
+9.  Confirm relationship between infrastructure index and indicator
+    values to justify the extrapolation
+
+10. Sample 10 00 random points from an ecosystem delineation and extract
+    indicator values and sd at those points.
+
+11. Export indicator maps and regional extrapolated maps
+
+## About the underlying data
+
+The indicator uses a data set from a standardised field survey of nature
+types. You can read more about the preliminary analyses
+[here](#naturtype). See also the [official
+site](https://www.miljodirektoratet.no/ansvarsomrader/overvaking-arealplanlegging/naturkartlegging/naturtyper/)
+of the Environmental Agency. I also import a data set called
+[ANO](#ANO-import), which you can read about
+[here](https://www.miljodirektoratet.no/ansvarsomrader/overvaking-arealplanlegging/miljoovervaking/overvakingsprogrammer/natur-pa-land/arealrepresentativ-naturovervakning-ano/)
+
+### Representativity in time and space
+
+The nature type mapping is not random and cannot be said to be area
+representative. The ANO dataset however, is area representative. The
+data is from 2018 to present. The data from one field season usually
+becomes available early the following year.
+
+### Original units
+
+The variables are recorded on a coarse four-sped ordinal scale (Fig.
+\@ref(fig:four-step)).
+
+### Temporal coverage
+
+The data goes back to 2018, but is because of data shortage, it is
+bulked into one time step. I then use the mean date for the raw data,
+and define the variable as belonging to the year 2020 (read more
+[here](#scaled-slitasje-variable)).
+
+### Aditional comments about the dataset
+
+For a run through of the data set, see [here](#naturtype).
+
+## Ecosystem characteristic
+
+### Norwegain standard
+
+The indicator is tagged to the *Økologisk egenskap* called
+**Primærproduksjon** (Primary productivity). This is tentative, and
+perhaps *abiotiske forhold* is more suited. But the thought behind the
+chiose is that *sliatsje* affect the potential for primary productivity.
+
+### UN standard
+
+The indicator is tagged as a **Physical state characteristics**
+indicator. This is quite clear. It's a tpe of abiotic characteritic that
+has to do with the physical structure rather than the chemical
+composition.
+
+## Collinearities with other indicators
+
+Slitasje is not thought to be very collinear with any other indicator at
```

Comment:
"... thought to exhibit collinearity with any other indicator.."

Reply:
good